Example/Config.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+import MiniApp
+
+class Config: NSObject {
+    enum Key: String {
+        case applicationIdentifier = "RASApplicationIdentifier",
+        version = "CFBundleShortVersionString",
+        subscriptionKey = "RASProjectSubscriptionKey",
+        endpoint = "RMAAPIEndpoint"
+    }
+
+    static let userDefaults = UserDefaults(suiteName: "com.rakuten.tech.mobile.miniapp.MiniAppDemo.settings")
```

Comment:
Could you initialize the class with a `UserDefaults` object, with default param set to what you've defined here?
I think it would make testing easier.

Reply:
Ah, it seems like you need this to be static.

Reply:
good idea, I'll implement this